docs/content/direct/release.md

```diff
@@ -38,16 +38,12 @@ Making a new kubestellar release requires a contributor to do the following thin
 
 - If not already in effect, declare a code freeze. There should be nothing but bug fixes and doc improvements while working towards a regular release.
 
-- Edit the source for the KCM PCH (in `config/postcreate-hooks/kubestellar.yaml`) and update the tag in the reference to the KCM container image (it appears in the last object, a `Job`).
+- Edit `docs/mkdocs.yml` and update the definition of `ks_latest_release` to `$version` (e.g., `'0.23.0-rc42'`). If this is a regular release then also update the definition of `ks_latest_regular_release`.
```

Comment:
As we discussed elsewhere - I think we have a notion of "latest" only for regular releases.

Reply:
To move the discussion into a public forum, created a GitHub "discussion". See https://github.com/kubestellar/kubestellar/discussions/2218 .